File: CS-3035-Programming-Paradigms/projects/lab22/lab22.py
def arpGenerator(score):
    if score < 500:
        return 23
    elif score < 600:
        return 11
    elif score < 700:
        return 8
    elif score < 800:
        return 5
    elif score < 851:
        return 3
    else: return -1;

score = input("What is your credit score? ")
print("Your ARP is " + str(arpGenerator(int (score))) + "%")
print("For 700 credit score, your APR would be " + str(arpGenerator(700)) + "%")
print("For 625 credit score, your APR would be " + str(arpGenerator(625)) + "%")

File: CS-3035-Programming-Paradigms/projects/lab22/lab22.hs
aprGenerator :: Int -> Int
aprGenerator a
    | a < 500 = 23
    | a < 600 = 11
    | a < 700 = 8
    | a < 800 = 5
    | a < 851 = 3
    | otherwise = -1 

main = do
    print( "For a credit score of 700, your APR would be "++ show ( aprGenerator 700 ) ++"%" )
    print( "For a credit score of 825, your APR would be "++ show ( aprGenerator 825 ) ++"%" )
